### PR TITLE
Better restaurants call

### LINKED ISSUE #15: Second API call - similar (better) locations
- [x] figure out what to go get - in category? Same star rating or above?
- [x] parse that from our current search result
- [x] package that up, new api call

### PR DESCRIPTION
**Pull request by bradleyjay**
Closes #15 

Added wrapper function for second API call to properly parse API call of Better Businesses, to then render in better_options.html

### CODE REVIEW

File: better_than_this/external/yelp_client.py [hunk 1]
```diff
@@ -13,22 +13,22 @@ def get_restaurant(name, address):
...
-            "id": candidate["id"],
-            "categories": candidate["categories"], # needs parsing fcn, API call will need comma seperated list
-            "price": candidate["price"], #may need to be converted: $ -> 1, $$ -> 2, etc with fcn
-            "request_duration": request_duration
-            }
+    return "ERROR - bad API response"
+
+def get_better_restaurants(search_result):
+    results, request_duration = businesses_search_suggestions(search_result)
+    # print(" **** THIS IS RESTAURANTS ***** \n" + str(restaurants))
```
Comment (aturkewi) [inline better_than_this/external/yelp_client.py]: get rid of

File: better_than_this/external/yelp_client.py [hunk 2]
```diff
@@ -13,22 +13,22 @@ def get_restaurant(name, address):
...
+    if results:
+        return list(map(location_from_api, results["businesses"]))
 
     return "ERROR - bad API response"
 
+
```
Comment (aturkewi) [inline better_than_this/external/yelp_client.py]: Get rid of these extra spaces

File: better_than_this/external/yelp_client.py [hunk 3]
```diff
@@ -66,7 +64,7 @@ def make_request(url):
     if response.status == 200:
         return decode_body_response(response), request_duration
     else:
-        return false
+        return False
```
Comment (aturkewi) [inline better_than_this/external/yelp_client.py:62]: NICE!

File: yelpAPItest.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+from better_than_this.external.yelp_client import businesses_search_suggestions
```
Comment (aturkewi) [inline yelpAPItest.py]: We should get rid of this file!

**Review by aturkewi — CHANGES_REQUESTED** [review]
Let's make a couple of changes first.

File: better_than_this/external/yelp_client.py [hunk 5]
```diff
@@ -13,19 +13,14 @@ def get_restaurant(name, address):
...
-            return {
-            "name": candidate["name"],
-            "address": parse_location(candidate["location"]),
-            "image_url": candidate["image_url"],
-            "id": candidate["id"],
-            "categories": candidate["categories"], # needs parsing fcn, API call will need comma seperated list
-            "price": candidate["price"], #may need to be converted: $ -> 1, $$ -> 2, etc with fcn
-            "request_duration": request_duration
-            }
+    return "ERROR - bad API response"
```
Comment (aturkewi) [inline better_than_this/external/yelp_client.py]:
```suggestion
    return "ERROR - API did not respond as intended"
```

**Review by aturkewi — APPROVED** [review]
👍

File: better_than_this/external/yelp_client.py (hunk 3 above)
Comment (bradleyjay) [reply better_than_this/external/yelp_client.py:62]: 👍

**Review by bradleyjay — COMMENTED** [review]
LGTM 👍

**Pull request merged**